protos/core.proto

```diff
@@ -82,7 +96,7 @@ message Table {
   string query = 5;
   bool protected = 9;
 
-  map<string, string> fieldDescriptor = 16;
+  ActionDescriptor action_descriptor = 24;
```

Comment:
as this is sometimes printed as json, would just calling the field `descriptor` be cleaner / sufficient?